src/view/emptyelement.js

```diff
@@ -29,6 +29,18 @@ export default class EmptyElement extends Element {
 		if ( arguments.length > 2 ) {
 			throwCannotAdd();
 		}
+
+		/**
+		 * Returns `null` because filler is not needed.
+		 * This method is implemented in a constructor because it is treated as an injected behaviour and it might be
+		 * changed during execution (in elements representing widgets for example).
+		 * When {@link module:engine/view/element~Element.clone element is cloned}, this method will be referenced from
```

Comment:
Unnecessary explanation from the API docs perspective (the whole piece about where it's defined and that it will be cloned). It's defined as a method, so it will be cloned and it will be available in all instances of this class.